builtin/providers/aws/provider_test.go

```diff
@@ -30,11 +30,13 @@ func TestProvider_impl(t *testing.T) {
 }
 
 func testAccPreCheck(t *testing.T) {
-	if v := os.Getenv("AWS_ACCESS_KEY_ID"); v == "" {
-		t.Fatal("AWS_ACCESS_KEY_ID must be set for acceptance tests")
-	}
-	if v := os.Getenv("AWS_SECRET_ACCESS_KEY"); v == "" {
-		t.Fatal("AWS_SECRET_ACCESS_KEY must be set for acceptance tests")
+	if v := os.Getenv("AWS_PROFILE"); v == "" {
```

Comment:
Is this the normal precedence of profile vs keys set explicitly? If so, this LGTM - if not we should probably reverse it to work the same way as the AWS CLI?